File: arrays-2.java
class Solution {
     static void  minmax(int[] arr) {
         if(arr == null || arr.length==0)return;
         int min=arr[0];
         int max=arr[0]
         for(int i=1 ; i <arr.length ; i++){
             min=Math.min(min,arr[i]);
             max=Math.max(max,arr[1]);
         }
         return min;
         return max;
        
    }
}

File: arrays2.cpp
pair<int,int> maxMin1(int arr[], int size){
    int max=INT_MIN;
    int min=INT_MAX;
    for(int i=0;i<size;i++){
        if(arr[i]>max){
            max=arr[i];
        }
        if(arr[i]<min){
            min=arr[i];
        }
    }
    return make_pair(max,min);
